model/yeast-GEM.txt

```diff
@@ -1497,7 +1497,7 @@ r_2101	9H-xanthine[e] <=> 9H-xanthine[c] 		-1000.00	1000.00	  0.00
 r_2102	xanthosine[e] =>  		  0.00	1000.00	  0.00
 r_2103	xanthosine[e] => xanthosine[c] 		  0.00	1000.00	  0.00
 r_2104	xylitol[e] =>  		  0.00	1000.00	  0.00
-r_2105	xylitol[e] <=> xylitol[c] 	YDL245C or YLL043W	-1000.00	1000.00	  0.00
```

Comment:
Double check this. According to [SGD](https://www.yeastgenome.org/locus/S000003966) and [Biocyc](https://biocyc.org/gene?orgid=YEAST&id=G3O-32144), YLL043W do has function for xylitol transport.